ibm_catalog.json

```diff
@@ -128,6 +128,9 @@
             },
             {
               "key": "topics"
+            },
+            {
+              "key": "metrics"
```

Comment:
can we customise this into a dropdown box with all the allowed values. The same way we have done for `plan` input

Reply:
The value is a list of strings, not a single string. It would need to be like a checkbox (multiselect), with the allowed values `topic`, `partition`, and `consumers`. I'm not familiar with the ibm-catalog spec, how would that be done?

- [ ] topic
- [ ] partition
- [ ] consumers

Reply:
ah sorry yes you are right - no action needed here then